src/gseproject/templates/game/QR code.html

```diff
@@ -16,9 +16,10 @@
 
 
 <script src="https://unpkg.com/html5-qrcode"></script>
-
-<script>
-    function domReady(fn) {
+<script src="path/to/base_game_template.js"></script>
```

Comment:
base_template.js should already be imported, making the line `<script src="path/to/base_game_template.js"></script>` redundant

Reply:
ok, should i delete that?

Reply:
yeah just drop that line